backend/app/lib/ark/archivesspace_ark_minter.rb

```diff
@@ -16,7 +16,7 @@ def mint!(obj, row_defaults)
 
   # True if the ARK has the right NAAN and has a number that falls within the
   # range that we have already minted.
-  def ark_recognized?(ark)
+  def ark_recognized?(ark, obj)
```

Comment:
I believe the `ark_recognized?` method is only there to support use cases where a user is updating existing arks for some reason, without using the minter. But would that ever be permissible in a NOID minter? I would think not. So I am wondering if this can just be left alone and the `ark_recognized?` can just use a fixed regex.